.env.example

```diff
@@ -1,4 +1,4 @@
-GOOGLE_API_KEY="Sample Key"
+GOOGLE_API_KEY="Sample Key "
```

Comment:
Why do you commit this?

Reply:
I need to update the branch first always.

Reply:
That doesn't make any sense

Reply:
is there any way out so that we can make development branch updated without deleting repo and re forking

Reply:
git pull -r upstream development

Reply:
Still 0 idea why this change is committed in all your PRs

Reply:
Thanks for guiding me, will be taking care of this from now.